app/views/users/settings/teams/new.html.erb

```diff
@@ -18,7 +18,7 @@
       </div>
 
       <div class="form-group" style="max-width: 500px;">
-        <%= f.text_area :description, label: t('users.settings.teams.new.description_label') %>
+        <%= f.text_area :description, class: "form-control", label: t('users.settings.teams.new.description_label') %>
```

Comment:
Does it must be 'sci-input-field' ?

Reply:
you need add `sci-input-container` instead of form-group

Reply:
It breaks the layout as sci-input-container are inline-box

Reply:
And what problem to fix layout?